src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    private boolean isDone = false;
+    public static int maxTaskNumber = 1;
+    private String taskName;
+
+    public String getDone() {
+        return isDone ? "X" : " ";
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public void setDone() {
+        isDone = true;
+    }
+
+    public void unsetDone() {
+        isDone = false;
+    }
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+    }
+
+    public void getTaskStatus() {
```

Comment:
Perhaps you could add some default behaviour to the function in this base class?